packages/manager/modules/error-page/README.md

```diff
@@ -0,0 +1,29 @@
+# manager-error-page
+
+[![npm version](https://badgen.net/npm/v/@ovh-ux/manager-error-page)](https://www.npmjs.com/package/@ovh-ux/manager-error-page) [![Downloads](https://badgen.net/npm/dt/@ovh-ux/manager-error-page)](https://npmjs.com/package/@ovh-ux/manager-error-page) [![Dependencies](https://badgen.net/david/dep/ovh-ux/manager/packages/manager/modules/error)](https://npmjs.com/package/@ovh-ux/manager-error-page?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/manager/packages/manager/modules/error)](https://npmjs.com/package/@ovh-ux/manager-error-page?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```

Comment:
For the `Dependencies` badge you have to replace `modules/error` by `modules/error-page` I think. (same for `Dev Dependencies`).